templates/app/index.html

```diff
@@ -6,7 +6,8 @@
     <title><%= name %></title>
     <meta name="description" content="<%= name %> : <%= description %>">
     <link rel="stylesheet" type="text/css" media="screen" href="styles/stylesheet.css">
-    <script src="scripts/offline-manager.js"></script>
+    <script data-worker="offliner-worker.js" src="lib/offliner/dist/offliner-client.min.js"></script>
+    <script defer src="scripts/index.js"></script>
```

Comment:
> Add a special data-autoinstall="true" to offliner to make it install the worker automatically.
> Lazy-load the library in the client code before installing.

I suggest the latter: lazy-load the library in the client. Then you don't have to change the behavior of *offliner* right now, and the client can conditionally load the library only if `'serviceWorker' in navigator`, which'll save a network request on clients that don't support it.